5EUNG/app.js

```diff
@@ -31,18 +31,89 @@ app.use(express.json());
 app.use(cors());
 app.use(morgan('dev'))
 
-
-// health check 
 app.get("/ping", (req, res) => {
   res.status(200).json({ message: "pong" })
 });
 
+app.post('/signup', async (req, res, next) => {
+  const { name, email, profileImage, password } = req.body
+
+  await appDataSource.query(
+    `INSERT INTO users(
+      name,
+      email,
+      profile_image,
+      password
+      ) VALUES (?, ?, ?, ?);
+    `,
+    [name, email, profileImage, password]
+  );
+
+  res.status(201).json({ message: "successfully signed up" });
+});
+
+app.post('/posts', async (req, res) => {
+  const { title, content, postImage, userId } = req.body
+
+  await appDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      post_image,
+      user_id
+      ) VALUES (?, ?, ?, ?);
+    `,
+    [title, content, postImage, userId]
+  )
+
+  res.status(201).json({ message: "postCreated" });
+})
+
+app.get('/posts/all', async (req, res) => {
```

Comment:
HTTP method 중 GET은 리소스를 조회하고 정보를 가져오는 역할을 합니다. 그렇기 때문에 모든 게시물의 정보를 가져오는 것은  `/posts`라는 엔드포인트를 사용하여도 게시글의 정보를 가져옴을 유추해볼 수 있습니다. 
이렇듯 자명한 엔드포인트를 작성하는 것에 대해 REST하게 엔드포인트를 작성한다고 합니다.
해당 내용은 이후에 있을 RESTful API 세션에서 다뤄질 예정이니 참고해주세요 :)
```suggestion
app.get('/posts/', async (req, res) => {
```